src/server/controllers/index.js

```diff
@@ -0,0 +1,84 @@
+const  express = require('express');
+var router = express.Router();
+
+/* Require model */
+var models = require('../models');
+/* Require middlewares functions */
+
+router.get('/', function (req, res) {
+    res.render('index.html', {title: "Interview Coding Challenge"});
+});
+
+router.get('/getAllFavStuffs', function (req, res) {
+    models.favorite_stuff.find({}, function (err, data) {
+        if (err) {
+            res.json({error: true, message: "There isn't information yet.", response: ""});
```

Comment:
the errors could be that the database service is not available or the collection does not exist, in this case, it is the only error because it is just returning all the information of that collection.
We can handle different errors on save, update, that the var types are not the same for example.